### DISCUSSION TITLE
No Public TestCase Data

### DISCUSSION THREAD

**Discussion by Realone24** [NASA-LIS/LISF#1080] [Q&A] [answered]
Hi,

I have been using public test cases to run the LIS, previously I downloaded it from NASA LIS webpage (https://lis.gsfc.nasa.gov/lis-testcases). However, it is now suggested to visit GitHub webpage (https://github.com/NASA-LIS/LISF/tree/master/docs/public_testcase_walkthrough).  But there is no link to download the testcase data set. Let me know where to download the data to run the all steps in testcases.

Thanks...

**Comment by karsenau** [comment, marked as answer]
Hi @Realone24:

Thank you for your interest in the LISF software.

To read the instruction documentation for the public LISF testcase available on our GitHub page, have you installed the ‘Asciidoctor.js Live Preview’ extension for either your Firefox or Chrome browsers?

Our documentation is written in AsciiDoc, and the mark up aspects require an extension or you can also do it offline with the Asciidoctor toolset (https://asciidoctor.org/).

If you haven't installed either of these for complete viewing of all the links in our docs, you can read about that more here at the top of our main documentation page: 
https://github.com/NASA-LIS/LISF/blob/master/docs/README.adoc

Please let us know if this helps to address your issue.

**Reply by bmcandr** [reply]
Thanks @karsenau! I'll add that the Public Testcase Walkthrough guide includes instructions for downloading the data in each step of the testcase. After installing the AsciiDoctor.js Preview browser extension, use this link to view the guide: https://raw.githubusercontent.com/NASA-LIS/LISF/master/docs/public_testcase_walkthrough/public_testcase_walkthrough.adoc
